### PR TITLE
Use golint

### PR DESCRIPTION
**Pull request by zehuaiWANG**
I have used golint and find some typo error and comment missing，and this is some of them.  It is only in backend/cert, backend/client and backend/resource/deployment. And I will do the others next time if it is necessary. @floreks @maciaszczykm

### CODE REVIEW

File: src/app/backend/cert/manager.go [hunk 1]
```diff
@@ -22,13 +22,13 @@ import (
 	certapi "github.com/kubernetes/dashboard/src/app/backend/cert/api"
 )
 
-// Implements cert/api/types.Manager interface. See Manager for more information.
+// Manager is used to implements cert/api/types.Manager interface. See Manager for more information.
```
Comment (maciaszczykm) [inline src/app/backend/cert/manager.go]: s/implements/implement

Comment (zehuaiWANG) [reply src/app/backend/cert/manager.go]: Do you mean to remove the 's' form "implements" ? I fix it now.

Comment (zehuaiWANG) [reply src/app/backend/cert/manager.go]: @floreks

Comment (zehuaiWANG) [reply src/app/backend/cert/manager.go]: @maciaszczykm

**Pull request merged**